### PR TITLE
Update to next version after 8.2.0 release

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

**Review by iNinja — APPROVED** [review]

File: test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj [hunk 1]
```diff
@@ -22,6 +22,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault.Cryptography" Version="$(MicrosoftAzureKeyVaultCryptographyVersion)" />
     <PackageReference Include="System.Text.RegularExpressions" Version="$(SystemTextRegularExpressions)"/>
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="$(MicrosoftRestClientRuntimeVersion)"/>
```
Comment (brentschmaltz) [inline test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj:25]: Was there a mistake in removing this package?

Comment (iNinja) [reply test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj:25]: I was thinking about the same. Removing the package did not remove the warnings/errors, apparently.

Comment (jennyf19) [reply test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj:25]: I added the package

Comment (jennyf19) [reply test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj:25]: We need to specify a certain version MS.Rest.ClientRuntime

**Review by msbw2 — APPROVED** [review]

**Review by westin-m — APPROVED** [review]

**Pull request merged**